### PR TITLE
Update manifests - inheritance no longer supported

### PR DESCRIPTION
**Pull request by lbeaufort**
Manifest inheritance and hosts have been deprecated in Cloud Foundry in favor of another type of setup using something called a YAML anchor.

Instead of inheriting manifest files from each other, you declare a defaults section in your manifest file that other sections then reference.  Unfortunately what this means for us is increased duplication since we have separate manifests for each space, but we want to make sure this is accounted for before the manifest inheritance is just shut off completely on us (this change happened about a year ago).

## Questions
- [x] OK?: feature: removed:
    `"/data": "https://fec-feature-web.app.cloud.gov",`
- [ ] OK?: Added `routes` to stage and feature

### CODE REVIEW

File: manifest_dev.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ---
-inherit: manifest_base.yml
+buildpack: staticfile_buildpack
+applications:
+  - name: proxy
+    memory: 64M
 routes:
-- route: fec-dev-proxy.app.cloud.gov
-#- route: transition.fec.gov
```
Comment (ccostino) [inline manifest_dev.yml]: Is transition.fec.gov no longer something we need to worry about, or just not for the `dev` environment?

Oh never mind, it was commented out anyway, so this is cleanup. :-)

File: manifest_feature.yml [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+    memory: 64M
+routes:
+  - route: fec-feature-proxy.app.cloud.gov
 services:
   - fec-creds-feature
 env:
   PROXIES: |
     {
       "/": "https://fec-feature-cms.app.cloud.gov",
-      "/data": "https://fec-feature-web.app.cloud.gov",
```
Comment (ccostino) [inline manifest_feature.yml]: Yup, no longer needed!  We don't have a separate `web` app anymore, so no need for this `/data` proxy rule. :-)

File: manifest_feature.yml [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
       "/regulations": "https://fec-feature-eregs.app.cloud.gov"
     }
+  S3_BUCKET_URL: "https://cg-449d4df6-4b9e-4539-891f-363302ca5907.s3-us-gov-west-1.amazonaws.com"
```
Comment (ccostino) [inline manifest_feature.yml]: This is the separate S3 bucket for the `feature` space, correct?

File: manifest_prod.yml [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
+applications:
+  - name: proxy
+    memory: 64M
 instances: 4
 routes:
-- route: beta.fec.gov
-- route: fec-prod-proxy.app.cloud.gov
-- route: transition.fec.gov
-- route: www.fec.gov
+  - route: beta.fec.gov
```
Comment (ccostino) [inline manifest_prod.yml]: Ah, we haven't deprecated `beta.fec.gov` yet, have we? :-)

File: manifest_dev.yml (hunk 1 above)
Comment (lbeaufort) [reply manifest_dev.yml]: @ccostino yep, thanks!

File: manifest_feature.yml (hunk 3 above)
Comment (lbeaufort) [reply manifest_feature.yml]: @ccostino I copied this over from the `manifest_base.yml` file.

File: manifest_prod.yml (hunk 4 above)
Comment (lbeaufort) [reply manifest_prod.yml]: @ccostino I don't think we have - looks like it still works. Maybe it's to redirect old links people saved when we were in beta?

File: manifest_feature.yml (hunk 2 above)
Comment (ccostino) [reply manifest_feature.yml]: Ah okay, I see, it is the same bucket as `dev`, which was intended when we set this back up.

Comment (ccostino) [reply manifest_feature.yml]: You know what, we don't even need this file - we don't run the `proxy` app in the `feature` space anymore. :-)

Comment (lbeaufort) [reply manifest_feature.yml]: @ccostino I'll remove it with a new commit.

**Comment by lbeaufort** [comment]
Removed `manifest_feature.yml` -  we don't run the `proxy` app in the `feature` space anymore.

**Pull request merged**